riptide-core/src/main/java/org/zalando/riptide/Http.java

```diff
@@ -56,8 +63,34 @@
     Requester execute(HttpMethod method, URI uri);
     Requester execute(HttpMethod method);
 
-    static HttpBuilder builder() {
+    static RequestFactoryStage builder() {
         return new DefaultHttpBuilder();
     }
 
+    interface RequestFactoryStage {
+        ConfigurationStage requestFactory(AsyncClientHttpRequestFactory requestFactory);
+        default ConfigurationStage simpleRequestFactory(final ExecutorService executor) {
```

Comment:
What about exposing an overload without any parameter i.e. `simpleRequestFactory()` ?

Reply:
I don't want to manage an executor service myself. Clients should take care of this, including stopping/shutting down.

Reply:
I'm actually thinking about moving the whole method to a static factory and out of the builder interface. But I have a couple of other PRs lined up and this will have to wait.